### PR TITLE
Related Campaigns Part 1 

### PR DESCRIPTION
**Pull request by Vmack**
### What's this PR do?

This pull request creates a reportback confirmation page and adds a new route with starter copy for admins

### How should this be reviewed?

👀

### Relevant tickets

References [Pivotal #173236800](https://www.pivotaltracker.com/story/show/173236800).

### Checklist

- [ ] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added screenshots of front-end changes on small, medium, and large screens.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: config/feature-flags.php [hunk 1]
```diff
@@ -23,4 +23,5 @@
     'refer_friends_incentive' => env('DS_ENABLE_REFER_FRIENDS_INCENTIVE', false),
     'sitewide_nps_survey' => env('DS_ENABLE_SITEWIDE_NPS_SURVEY', false),
     'volunteer_credits' => env('DS_ENABLE_VOLUNTEER_CREDITS', false),
+    'post_confirmation_page' => env('DS_ENABLE_POST_CONFIRMATION_PAGE', false),
```
Comment (katiecrane) [inline config/feature-flags.php:26]: Let's add this variable to the `.env.example` file too!

Comment (Vmack) [reply config/feature-flags.php:26]: @katiecrane yup, it's there!

**Comment by katiecrane** [comment]
hmm @Vmack I set the `DS_ENABLE_POST_CONFIRMATION_PAGE` environment variable to `true` on the review app but when I visit `/us/posts/1822` (just picking a random post id) I get a `404` "Not Found" page. Are you able to hit that path locally? Or do I have the wrong one?

**Comment by Vmack** [comment]
@katiecrane hmmm that's strange...the path works for me. It looks like your configuration is the same, maybe try refreshing.

**Comment by Vmack** [comment]
@katiecrane 
Screen Shot 2020-10-13 at 9 06 41 AM

**Comment by katiecrane** [comment]
@Vmack this is so spooky! I'm re-creating the review app to see if anything changes, just to check!

**Comment by katiecrane** [comment]
it's working on the review app!! 
image

and confirmed that I am not able to visit the page without `DS_ENABLE_POST_CONFIRMATION_PAGE` set to true, so that is working as well!

**Pull request merged**